### ISSUE TITLE
Virginia

### ISSUE THREAD

**Issue by prikhi** [covid-db/covid-scrapers#16]
http://www.vdh.virginia.gov/coronavirus/

In the `Data Download section`, the `Download CSV version of VDH COVID-19-PublicUseDataset-Cases` link points to a CSV at http://www.vdh.virginia.gov/content/uploads/sites/182/2020/03/VDH-COVID-19-PublicUseDataset-Cases.csv

**Comment by mightybyte** [comment]
Yes, Virginia's CSV's look very good.  We should scrape everything they have and make some small tweaks to get field names into a canonical form, etc.

**Comment by mightybyte** [comment]
Data from the other ticket.

cases reported 1020
http://www.vdh.virginia.gov/

from county cases
http://www.vdh.virginia.gov/coronavirus/
more specific
https://public.tableau.com/shared/CDSX4BH9Q?:display_count=n&:origin=viz_share_link&:embed=y
